overlord/fdestate/backend/reseal.go

```diff
@@ -161,13 +177,16 @@ func resealRunObjectKeys(updateState StateUpdater, pbc boot.PredictableBootChain
 		return fmt.Errorf("cannot reseal the encryption key: %v", err)
 	}
 
```

Comment:
there is no resealing here based on pcrProfileRunOnly? do we need a TODO here? at least the title of the PR seems a bit misleading, as we compute a profile for run-only but don't apply it to any key atm

Reply:
Correct. Because we have no such key. There is some refactoring needed. First step will be https://github.com/canonical/snapd/pull/14674
That will split the calculation of the state from the actual resealing. Then we will have to change we reseal keys by discovering the key datas and what type they are.

Reply:
can we change the title and the commit as there is no reseal happening yet so I find it very confusing?